backend/src/core/loaders/sequelize.loader.ts

```diff
@@ -3,7 +3,7 @@ import { Sequelize, SequelizeOptions } from 'sequelize-typescript'
 import config from '@core/config'
 import { initializeModels } from '@core/models'
 
-import { loggerWithLabel } from '@core/logger'
+import { loggerWithLabel } from '@shared/core/logger'
```

Comment:
I rmb intentionally left this out of the shared folder when I was doing some refactoring previously cos they loggers are not the same

Reply:
Could you double check?